knex.js

```diff
@@ -3,20 +3,33 @@ const knex = require('knex') ({
   connection: 'postgres://localhost:5432/to-do'
 })
 
-const add = function(taskObject) {
+function add(taskObject) {
```

Comment:
You can just name the parameter `task`. In general, we don't want to mix business jargon with coding jargon.